### DISCUSSION TITLE
Betreiberverordnung umbenennen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#16] [Shipped - Live on production]
Betreiberverordnung ist nicht ganz eindeutig bzw. selbsterklärend. Nur in DE relevant. Umbenennen in "Anlage MPBetreibV"?

**Comment by YvesR** [comment]
Where?

**Reply by moritzvolkmar** [reply]
Device model catalog and inventory grid

![Bildschirm­foto 2023-01-30 um 15 22 20](https://user-images.githubusercontent.com/80519295/215503097-f76e2ce3-5fff-4923-b7f9-8448e62dccaf.png)

**Comment by YvesR** [comment]
We can rename it, but then we hide MPBetreibV fields (Annex 1+2) completely for all non German-Tenants?

**Reply by moritzvolkmar** [reply]
This would also be my suggestion
